splunk-otel-android/src/main/java/com/splunk/rum/RumInitializer.java

```diff
@@ -120,6 +127,20 @@ SplunkRum initialize(Supplier<ConnectionUtil> connectionUtilSupplier, Looper mai
         return new SplunkRum(openTelemetrySdk, sessionId, config);
     }
 
+    private SlowRenderingDetector buildSlowRenderDetector(Config config, Tracer tracer) {
```

Comment:
there's some broken parallel naming here. is it a "slow render detector" or a "slow rendering detector"?

Reply:
I can make it match the class name. 👍🏻